### ISSUE TITLE
SC2240 should be labeled as SC2169

### ISSUE THREAD

**Issue by nihilus** [koalaman/shellcheck#1384]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [x] I read the issue's wiki page, e.g. https://github.com/koalaman/shellcheck/wiki/SC2086
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

The problem here is that busybox's dash implementation actually supports this.

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
. /etc/script foo bar

```

#### Here's what shellcheck currently says:
SC2240: The dot command does not support arguments in sh/dash. Set them as variables.

#### Here's what I wanted or expected to see:
Something related to SC2169.

**Comment by koalaman** [comment]
It's more likely to go the other way as suggested by #503. Everything that's currently SC2169 should have its own error code. This will allow more fine-grained control over which features you do and don't want to allow.
